public/422.html

```diff
@@ -1,67 +1,106 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title>The change you wanted was rejected (422)</title>
+  <title>Server Error</title>
   <meta name="viewport" content="width=device-width,initial-scale=1">
   <style>
-  .rails-default-error-page {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
-
-  .rails-default-error-page div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
-
-  .rails-default-error-page div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-
-  .rails-default-error-page h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
-
-  .rails-default-error-page div.dialog > p {
-    margin: 0 0 1em;
-    padding: 1em;
-    background-color: #F7F7F7;
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #999;
-    border-bottom-left-radius: 4px;
-    border-bottom-right-radius: 4px;
-    border-top-color: #DADADA;
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+    @import url(https://fonts.googleapis.com/css?family=Gilda+Display);
+
+    html {
+      background: radial-gradient(#000, #111);
+      color: white;
+      overflow: hidden;
+      height: 100%;
+      user-select: none;
+    }
+
+    .error {
+      text-align: center;
+      font-family: 'Gilda Display', serif;
+      font-size: 95px;
+      font-style: italic;
+      text-align: center;
+      width: auto;
+      height: 60px;
+      line-height: 60px;
+      margin: auto;
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: -60px;
+      right: 0;
+      animation: noise-1 2s linear infinite;
+      overflow: default;
+    }
+
+    .info {
+      text-align: center;
+      font-family: 'Gilda Display', serif;
+      font-size: 15px;
+      font-style: italic;
+      text-align: center;
+      width: 400px;
```

Comment:
The only real difference between these error pages is the .info{width} as the messages are longer